flocker_bb/builders/maint.py

```diff
@@ -212,4 +235,6 @@ def getSchedulers():
     # This is so the zulip reporter gives better message.
     hourly.codebases = {'maint': {'branch': 'hourly'}}
 
+    # FLOC-2288 Schedule the clean-old-volumes hourly
```

Comment:
It was objectively easier not to have to introduce a new scheduler by re-using the existing builder, huzzah.